forms/common-themes/src/main/resources/theme/base/admin/resources/partials/password-policy.html

```diff
@@ -30,7 +30,7 @@
             <tr ng-repeat="p in policy">
                 <td>{{p.name|capitalize}}</td>
                 <td>
-                    <input class="form-control" ng-model="p.value" ng-show="p.name != 'notUsername' "
+                    <input id="{{p.name + (p.name == 'regexPattern' && '_' + $index || '')}}" class="form-control" ng-model="p.value" ng-show="p.name != 'notUsername' "
```

Comment:
Isn't the way for getting the input nicer and more robust with something like `input[starts-with(@id,"regex")]` than `tr/td[contains(text(),"RegexPattern")]/following-sibling::td/input`?
The index in the ID attribute would be for uniqueness of the attribute not for ordering.